prisma/schema.prisma

```diff
@@ -2,53 +2,55 @@
 datasource db {
     provider = "postgresql"
     url      = env("DATABASE_URL")
+    shadowDatabaseUrl = env("SHADOW_DATABASE_URL")
 }
 
 generator client {
-    provider        = "prisma-client-js"
+    provider = "prisma-client-js"
 }
 
-// model Post {
-//     id        String  @id @default(cuid())
-//     title     String
-//     content   String?
-//     published Boolean @default(false)
-//     author    User?   @relation(fields: [authorId], references: [id])
-//     authorId  String?
-// }
-
-// model User {
-//     id        String   @id @default(cuid())
-//     name      String?
-//     email     String?  @unique
-//     createdAt DateTime @default(now()) @map(name: "created_at")
-//     updatedAt DateTime @updatedAt @map(name: "updated_at")
-//     posts     Post[]
-
-//     @@map(name: "users")
-// }
 model Category {
-    id    Int    @id @default(autoincrement())
-    name  String @db.VarChar(30)
-    icon  String @db.VarChar(30)
-    color String @db.Char(7)
+    id       Int       @id @default(autoincrement())
+    name     String    @db.VarChar(30)
+    icon     String    @db.VarChar(30)
+    color    String    @db.Char(7)
+    articles Article[]
```

Comment:
Why do we need an article array inside category? To fetch it per category and show on page?

Reply:
I'll have to check in why, however when I entered property `Category` on entity `Article`, my IDE was flagged with error where I added this line 17.